### DISCUSSION TITLE
IKEA E2002 not supported but covered in supported devices list.

### DISCUSSION THREAD

**Discussion by dirkalferink** [Koenkk/zigbee2mqtt#8151] [Devices]
I'm trying to pair the Ikea E2002, however, zigbee2mqtt UI in homeassistant mentiones that the device is not supported while the list says it is.

see device here: https://www.zigbee2mqtt.io/devices/E2001_E2002.html

**Comment by Koenkk** [comment]
Make sure you are using the latest z2m version.

If you are still on 1.18.1 check first point of https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.19.0

**Reply by robi59italy** [reply]
your suggestion solved my problems ! very good
